app/controllers/users_controller.rb

```diff
@@ -13,6 +16,10 @@ def create
     end
   end
 
+  def edit
+    @user = User.find(params[:id])
+  end
+
   private
 
```

Comment:
We need to include the update action here. It redirects to edit_user_path if the user was saved (updated) successfully. And also render `:edit `if the user can't be saved(updated)